### DISCUSSION TITLE
Dark mode?

### DISCUSSION THREAD

**Discussion by jackuars7** [filecxx/FileCentipede#549] [Ideas]
Hello sir, 

I have very sensitive eyes and so most of the apps and softwares that I use are in dark mode. I hope FileCentipede also supports dark mode as this has become universal for almost all softwares. 

Regards,
Geejo Joy

**Comment by filecxx** [comment]
https://github.com/filecxx/FileCentipede/issues/324
https://github.com/filecxx/FileCentipede/issues/467

I'll add the dark mode theme after FileCentipede3.0.

**Comment by jackuars7** [comment]
Thank you for your response. The only reason why FileCentipede is not my daily driver is that I perform a lot of downloads in a day, [which  means a lot of screen time] and I have to use an alternate download manager for the same because of its dark mode. 

Waiting for 3.x version to happen soon. Thanks again for your hard work.

**Comment by Krezzin** [comment]
TFW the Dark Mode support may come on Judgment Day... Hehehe.
